src/components/@common/Recommendations/User/UserCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * 친구 찾아보기 - 유저 카드
+ * @author 선우
+ */
+
+import PersonAdd from '@/components/@common/SVG/Icon/PersonAdd';
+import { IRecommendedUsers } from '@/types/@common';
+import { ReactEventHandler, useState } from 'react';
+import * as Styled from './UserCard.styled';
+import defaultProfileImg from '@/assets/images/defaultProfile.png';
+
+export interface IUserCardProps {
+  userInfo: IRecommendedUsers['content'][0];
```

Comment:
```suggestion
  userInfo: IProfileSimpleResponse;
```
이미 정의된 IProfileSimpleResponse로 사용해도 좋을 것 같습니다.

Reply:
피드백 감사합니다 🙇

해당 타입이 이미 존재하고 있었네요! 😅 
말씀 주신대로 생성한 type은 삭제하고 **`IProfileSimpleResponse`** 타입을 이용할 수 있도록 수정했습니다.